Tests/MapKitTests/MKMapViewTests.swift

```diff
@@ -35,7 +35,41 @@ final class MKMapViewTests: XCTestCase {
         let annotationViewWithAnnotation = mapView.dequeueReusableAnnotationView(withClass: MKPinAnnotationView.self, for: annotation)
         XCTAssertNotNil(annotationViewWithAnnotation)
     }
+    
+    func testWithEmptyItemArray() {
+        let mapView = MKMapView()
+        let meter = 500.0
+        let emptyItemArray = [CLLocationCoordinate2D]()
+        let edgePadding = UIEdgeInsets(top: 50, left: 50, bottom: 50, right: 50)
+        
+        let emptyItemInMapView: Void = mapView.zoom(to: emptyItemArray, meter: meter, edgePadding: edgePadding, animated: true)
```

Comment:
Hey @mustafagunes, this test still seems incorrect, consider using the same logic that you are using in `testWithMultiItemArray` and `testWithOneItemArray` :)